README.md

```diff
@@ -1,100 +1,58 @@
 
-### Github Package dependencies
-In order to pull in Github package dependencies you will need a Github Personal Access Token.
-This token will need the minimum of 'packages:read' permissions.
-
-Update your .m2/settings.xml file to contain the <servers><server> tags like timecard_settings.xml
-The token will need to live within your local .m2/settings.xml file as the password
-
-For more info see:
-[https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry)
-
-Then run the following to build the project
-
-```sh
-$ mvn clean install
-```
+## Running as part of the LocalDev environment
 
-### Create database
+1. Download LocalDev repository from https://github.com/UKHomeOffice/callisto-localdev and run it locally as described in [Scenario 1](https://github.com/UKHomeOffice/callisto-localdev#scenario-1-running-callisto-without-need-to-edit-code-base-eg-demo-purposes).
 
-If you need to create your own postgres database you can use docker. This works well if you create a container and name it, then you can stop and start it as you please
+2. From the LocalDev project root, stop the service by running `docker compose stop <service_name>` command.
 
-```sh
-docker create --name postgres-local -e POSTGRES_PASSWORD=Welcome -p 5432:5432 postgres:11.5-alpine
-docker start postgres-local
-```
+3. Ensure you have configured your [Authentication for GitHub Packages](#authentication-for-github-packages)
 
-Connect to database using `psql`
+4. Pull the service repository, and from its root directory run command `docker compose up -d`
 
-```sh
-psql -h [YOUR DOCKER HOST IP ADDRESS] -U postgres
-```
+After successful start, you should be able to work with the service code, and see all changes without restaring the docker container.
 
-For example if `echo $DOCKER_HOST` returns `tcp://192.168.64.4:2376` then run `psql -h 192.168.64.4 -U postgres`
-
-If the above echo command doesn't work try:
-```sh
-docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' postgres-local
-```
-Create the database
-```sql
-CREATE DATABASE callisto;
-```
+## Authentication for Github Packages
 
-### Setting environment variables 
+In order to pull in Github package dependencies you will need a Github Personal Access Token.
+This token will need the minimum of 'packages:read' permissions.
 
-Before running the database locally, you need to set your local environment variables e.g.
+Update your .m2/settings.xml file to contain the <servers><server> tags
 
-```
-DATABASE_ENDPOINT=localhost
-DATABASE_NAME=callisto
-DATABASE_PORT=5432
-DATABASE_USERNAME=postgres
-DATABASE_PASSWORD=Welcome
+```xml
+<servers>
+  <server>
+      <id>github-packages</id>
+      <username>DummyUser</username>
+      <password>${env.GITHUB_TOKEN}</password>
```

Comment:
```suggestion
      <password>{[Insert GitHub Personal Access Token here]}</password>
```

Reply:
I did want the reader to be encouraged to inject their token insecurely by an environment variable instead of encrypting it.